__tests__/bubbleNub.js

```diff
@@ -28,10 +28,12 @@ describe('bubble nub', () => {
     const { driver, pageObjects } = await setupWebDriver({
       props: {
         styleOptions: {
-          bubbleNubOffset: 0,
```

Comment:
You should be adding new tests instead of modifying the old ones.

Reply:
Noted!